floof.yaml

```diff
@@ -5,6 +5,37 @@ default:
     - run-task: frontend
     - run-task: backend
 
+# Starts all services from `scripts`
+start-scripts:
+  - set-workdir: scripts
+  - concurrently:
+    - docker-compose -f auth-proxy/docker-compose.yml up -d
+    - docker-compose -f dev-db/docker-compose.yml up -d
+    - docker-compose -f login-handler/docker-compose.yml up -d
+    - docker-compose -f opencast-cors/docker-compose.yml up -d
+
+# Stop all those servives again
+stop-scripts:
+  - set-workdir: scripts
+  - concurrently:
+    - docker-compose -f auth-proxy/docker-compose.yml down
+    - docker-compose -f dev-db/docker-compose.yml down
+    - docker-compose -f login-handler/docker-compose.yml down
+    - docker-compose -f opencast-cors/docker-compose.yml down
```

Comment:
Note that `down` is different from `stop`. Down also removes volumes, etc. Is that what we want here?